validation-api/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.11-slim
+
+WORKDIR /app
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+EXPOSE 5000
+ENV PYTHONPATH="${PYTHONPATH}:/app"
+CMD ["flask", "--app", "api/app.py", "run", "--host=0.0.0.0"]
```

We've had somewhat inexplicable issues with working directories and Azure Batch. Things run fine locally but in Batch the working directory of the container is moved to somewhere other than `/`. Batch persists with this change despite `WORKDIR` in the `Dockerfile` and/or setting the working directory in a script. It's been frustrating because the errors aren't reproducible in the container locally, where moving the working directory has been fine.

So perhaps this might be safer?
```suggestion
CMD ["flask", "--app", "/api/app.py", "run", "--host=0.0.0.0"]
```
I wanted to test locally that this wasn't breaking things, but can't because of the above error.

---

Ok running `docker-compose up` from `validation-api/` got things working. The above suggestion didn't break things.

```
curl localhost:5000
> <p>Config validation server running.</p>
```

---

Updated to absolute path.